backends/gdx-backend-robovm/src/com/badlogic/gdx/backends/iosrobovm/IOSInput.java

```diff
@@ -639,7 +640,13 @@ private int findPointer (UITouch touch) {
 		for (int i = 0; i < touchDown.length; i++) {
 			if (touchDown[i] == ptr) return i;
 		}
-		throw new GdxRuntimeException("Couldn't find pointer id for touch event!");
+		// If pointer is not found
+		StringBuffer buf = new StringBuffer();
```

Comment:
Did you mean `StringBuilder`?

Reply:
True, I just copied AndroidInput code :), will create a small PR for it and fix this

Reply:
Ok, changes made and squashed, thanks!